### ISSUE TITLE
[BUG]Update failed for AnAction(com.intellij.ide.actions.ExternalJavaDocAction, id=ExternalJavaDoc) with text=External Documentation

### ISSUE THREAD

**Issue by PiryazevAlex** [flikas/archived-idea-spring-boot-assistant#18]
## What happened
> Please describe what you were doing when this exception occurred.

## Expected behavior
> A clear and concise description of what you expected to happen.

## Screenshots
> If applicable, add screenshots to help explain your problem.

## Version information
- OS: `Mac OS X 12.4 aarch64`
- IDE: `IntelliJ IDEA 2022.1.1(IU-221.5591.52)`
- Plugin: `Spring Boot Assistant 0.14.0`

## Exception
update failed for AnAction(com.intellij.ide.actions.ExternalJavaDocAction, id=ExternalJavaDoc) with text=External Documentation

<details><summary>Stack trace</summary>

```
java.lang.NullPointerException
	at in.oneton.idea.spring.assistant.plugin.suggestion.completion.YamlDocumentationProvider.getCustomDocumentationElement(YamlDocumentationProvider.java:71)
	at com.intellij.lang.documentation.DocumentationProvider.getCustomDocumentationElement(DocumentationProvider.java:189)
	at com.intellij.lang.documentation.CompositeDocumentationProvider.getCustomDocumentationElement(CompositeDocumentationProvider.java:291)
	at com.intellij.codeInsight.documentation.DocumentationManager.customElement(DocumentationManager.java:994)
	at com.intellij.codeInsight.documentation.DocumentationManager.doFindTargetElementAtOffset(DocumentationManager.java:972)
	at com.intellij.codeInsight.documentation.DocumentationManager.findTargetElementAtOffset(DocumentationManager.java:958)
	at com.intellij.codeInsight.documentation.DocumentationManager.findTargetElementUnsafe(DocumentationManager.java:948)
	at com.intellij.codeInsight.documentation.DocumentationManager.findTargetElement(DocumentationManager.java:931)
	at com.intellij.codeInsight.documentation.DocumentationManager.findTargetElement(DocumentationManager.java:925)
	at com.intellij.ide.actions.ExternalJavaDocAction.getElement(ExternalJavaDocAction.java:137)
	at com.intellij.ide.actions.ExternalJavaDocAction.update(ExternalJavaDocAction.java:147)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performDumbAwareUpdate$0(ActionUtil.java:131)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:145)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.doUpdate(ActionUpdater.java:645)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$updateActionReal$4(ActionUpdater.java:126)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.callAction(ActionUpdater.java:176)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.updateActionReal(ActionUpdater.java:127)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$new$0(ActionUpdater.java:112)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.update(ActionUpdater.java:634)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandGroupChild(ActionUpdater.java:499)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$doExpandActionGroup$20(ActionUpdater.java:457)
	at com.intellij.util.containers.ContainerUtil.concat(ContainerUtil.java:1400)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.doExpandActionGroup(ActionUpdater.java:457)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandActionGroup(ActionUpdater.java:260)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$12(ActionUpdater.java:321)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$13(ActionUpdater.java:339)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1152)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$tryRunReadActionAndCancelBeforeWrite$17(ActionUpdater.java:371)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:158)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.tryRunReadActionAndCancelBeforeWrite(ActionUpdater.java:367)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$14(ActionUpdater.java:339)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:189)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:608)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:683)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:639)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:607)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:176)
	at com.intellij.openapi.progress.util.BackgroundTaskUtil.runUnderDisposeAwareIndicator(BackgroundTaskUtil.java:365)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.lambda$expandActionGroupAsync$15(ActionUpdater.java:338)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:241)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:31)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:214)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:212)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:203)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
```
</details>

## Additional context
> Add any other context about the problem here.

**Comment by flikas** [comment]
duplicate of #16

**Comment by flikas** [comment]
Fixed by #17
